tests/test_tpfmachine.py

```diff
@@ -86,3 +89,73 @@ def test_parse_TPFs():
 def test_from_TPFs():
     c = TPFMachine.from_TPFs(tpfs)
     assert isinstance(c, Machine)
+
+
+def test_do_tiled_query():
+    # unit test for TPF stack
+    # test that the tiled query get the same results as the original psfmachine query
+    epoch = Time(tpfs[0].time[len(tpfs[0]) // 2], format="jd").jyear
+    sources_org = _get_coord_and_query_gaia(tpfs, magnitude_limit=18, dr=3)
+    _, ra, dec = _wcs_from_tpfs(tpfs)
+    sources_tiled = do_tiled_query(
+        ra,
+        dec,
+        ngrid=(2, 2),
+        magnitude_limit=18,
+        dr=3,
+        epoch=epoch,
+    )
+    assert isinstance(sources_tiled, pd.DataFrame)
+    assert set(["ra", "dec", "phot_g_mean_mag"]).issubset(sources_tiled.columns)
+    assert sources_tiled.shape == (83, 11)
```

Comment:
We might change these test files, or something might go awry. 
```suggestion
    assert sources_tiled.shape == sources_org.shape
```

Reply:
the tiled query will always return more sources compared to the original query function for a TPF stack. The tiled query is not optimized for sparse sky patches. This is why I am testing that the tiled query **contains** the results of the original query. They won't have the same shape.

Reply:
If we do `sources_tiled.shape == sources_org.shape` then we have to run `_clean_source_list()` for both results first